doc/classes/ScriptExtension.xml

```diff
@@ -80,6 +80,12 @@
 			<description>
 			</description>
 		</method>
+		<method name="_get_script_method_argument_count" qualifiers="virtual const">
```

Comment:
This now expects either an `int` if valid, or `null` if not, will see about documenting but this file is sparse already so will see

Reply:
`Return the expected argument count for the given [param method], or [code]null[/code] if it can't be determined.`

I've seen new methods added here many times, but they never get a description because "the class is not documented anyway".

Reply:
I'll document it in the extension case, it doesn't have it doesn't actually handle `null` special rn, will look though

Reply:
I'll add it tomorrow, or in the follow up PR, someone (not me) should document all of these though